supervisor/api/__init__.py

```diff
@@ -258,6 +258,14 @@ def _register_os(self) -> None:
             ]
         )
 
+        # Config options
+        self.webapp.add_routes(
+            [
+                web.get("/os/config/swap", api_os.config_swap_info),
+                web.post("/os/config/swap", api_os.config_swap_options),
```

Comment:
Obviously more of a nitpick but we have `/os/config/sync` up above, think we should just add these two to the list up here:
https://github.com/home-assistant/supervisor/blob/c2455f892619facbda38a679d1638b7bbe341408/supervisor/api/__init__.py#L242